RELEASE/scripts/autoscend.ash

```diff
@@ -1203,7 +1203,21 @@ void initializeDay(int day)
 			cli_execute("counters");
 			council();
 		}
-	}
+		
+		// If we have the shortest order cook, loop familiars that will benefit from that.
+		if(pathHasFamiliar() && pathAllowsChangingFamiliar()) {
+			familiar init_fam = my_familiar();
+			if (have_familiar($familiar[shorter-order cook]))
+			{
+				if (have_familiar($familiar[ghost of crimbo carols])) {
+					use_familiar($familiar[ghost of crimbo carols]);
+					use_familiar($familiar[ghost of crimbo commerce]);
+					use_familiar($familiar[ghost of crimbo cheer]);
+				}
+			}
+			use_familiar(init_fam);
+		}
```

Comment:
```suggestion
		if (pathHasFamiliar() && pathAllowsChangingFamiliar())
		{
			familiar init_fam = my_familiar();
			if (have_familiar($familiar[shorter-order cook]))
			{
				foreach fam in $familiars[ghost of crimbo carols, ghost of crimbo commerce, ghost of crimbo cheer]
				{
					if (have_familiar(fam))
					{
						use_familiar(fam);
					}
				}
			}
			use_familiar(init_fam);
		}
```

That way it won't fall over if the user hasn't added all 3 familiars to their terrarium and can be easily extended for other cases if any occur in existing or future IotMs.
Also fixed the inconsistent formatting.

Reply:
Great point on the "not used all familiars" and making it more flexible for other additions to the roster. I'd be fine with the committing that as is but regarding your point about refactoring, I agree this might not be best option.

This could be split out in a couple of ways. One, suggested by Beldur, for familiar initialisation, which could do all sorts of stuff, like free familiar equipment (bugbear, gnome). The other being placing those specific functions into their year IOTM files, as you suggested. The smartest thing to do would be probably both?